File: dev-docs/snippets/document-blob/resume-upload.ts
import type { JsBaoClient } from "js-bao-wss-client";

export function resumeUpload(
  client: JsBaoClient,
  documentId: string,
  blobId: string,
) {
  // #region example
  const blobs = client.document(documentId).blobs();
  const resumed = blobs.resumeUpload(blobId); // true if a paused upload was resumed
  // #endregion example
  return resumed;
}

File: dev-docs/snippets/document-blob/resume-upload.swift
import JsBaoClient

func resumeUpload(
  client: JsBaoClient,
  documentId: String,
  blobId: String
) {
  // #region example
  let blobs = client.documents.blobs(documentId: documentId)
  let resumed = blobs.resumeUpload(blobId: blobId)  // true if a paused upload was resumed
  // #endregion example
  _ = resumed
}
